scripts/perftune.py

```diff
@@ -1366,7 +1384,10 @@ def dump_config(prog_args):
 
 # set default values #####################
 if not args.nic:
```

Comment:
I don't think args.nic can ever be not None after you changed the `dest` to be `args.nics`.
I think you should drop this whole if-then. You already have the corresponding handling of args.nics right below.